python/zygote.py

```diff
@@ -68,8 +69,8 @@
 def try_dumps(obj, sort_keys=False, allow_nan=False):
     try:
         return json.dumps(obj, sort_keys=sort_keys, allow_nan=allow_nan)
-    except:
-        print("Error converting this object to json:\n{:s}\n".format(str(obj)))
+    except TypeError:
```

Comment:
Although this is a "less general" exception catch, this function just prints the error and re-raises an exception, so nothing should break from this (not that I expect it to anyway).

Reply:
I think I'd rather keep this as general as possible so that *any* serialization error results in the object being displayed. I think that's pretty crucial for being able to debug errors.

Reply:
Done!